docs/manual/macros.md

```diff
@@ -121,6 +121,7 @@ various common operations.
 | `%{lower:...}`   | lowercase a string | 4.19.0
 | `%{rep:...}`     | repeat a string (see Lua `string.rep()`) | 4.19.0
 | `%{reverse:...}` | reverse a string | 4.19.0
+| `%{span:...}`    | as-is string, handy for wrapping multiline macros | 6.0.0
```

Comment:
In light of the above discussion, doesn't the "as-is string" makes you think the string is in fact *literal* (i.e. with macros expanded)?

Reply:
> ...the %1 inside the span will get expanded when _%foo() is _used_

Yep, that's the confusing part since the doc says about the body of that macro that it's an "as-is string", whereas what it does is specifically discard newlines :dizzy_face: 

> Confusing enough? 😄

Just a little bit :smile:

Reply:
But the name actually implies that it's about *spanning* multiple lines, so that's fine, it's just the description that trips me up.

Reply:
OK, actually looking at the description in the *context* of the whole document (or at least the "String operations" table where this is located), it clicks now for me. As in, all the other string operations do some transformations so in that sense, "as-is" means "no transformations whatsoever". That's actually sensible.